### DISCUSSION TITLE
Dice Score computation

### DISCUSSION THREAD

**Discussion by ajulyav** [Project-MONAI/tutorials#989] [Q&A] [answered]
Hello,

The computation of Dice Loss and Dice Score bothers me. 
In particular, I found in literature that Dice Score can be computed as 1 - Dice Loss.

So, I see that Monai implemented Dice Score as well. But when I try to compare Dice (from Monai) and Dice as (1 - Dice Loss), they do not match. I expect that Dice Loss + Dice Metric should sum to 1? So, it does not work well in my case.

What could cause this difference? 

I took a code from one of the examples provided by Monai:
```

loss = DiceLoss(sigmoid=True)
transform_post = Compose([EnsureType(), Activations(sigmoid=True), AsDiscrete(threshold=0.5)])
inference = SlidingWindowInferer(roi_size=infer_roi_size, sw_batch_size=4, overlap=0.25)
valid_metric = DiceMetric(include_background=True, reduction="mean", get_not_nans=False)

test_outputs = inference(test_volume, model)
test_outputs = transform_post(test_outputs)

test_loss = loss(test_outputs, test_label)

print("loss : " + str(test_loss.item()))

# Method 1 (Monai)
test_metric = valid_metric(test_outputs, test_label)
print("Dice Metric : " + str(test_metric.item()))

# Method 2 (Dice = 1 - Dice loss)
test_metric= 1 - test_loss
print("Dice as 1 - Dice loss : " + str(test_metric))
```

So, here is my output:

loss : 0.9571341276168823
Dice Metric : 0.10580694675445557
Dice as 1 - Dice loss : 0.042865872383117676

loss : 0.9420668482780457
Dice Metric : 0.13196243345737457
Dice as 1 - Dice loss : 0.057933151721954346

**Comment by wyli** [comment, marked as answer]
(in case you haven't seen this, there's a tutorial on this topic https://github.com/Project-MONAI/tutorials/blob/main/modules/dice_loss_metric_notes.ipynb)
